keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -275,23 +277,25 @@ I want the ability to add a previously-isolated cluster to a ClusterSet, or to m
 
 I have a headless multi-cluster service deployed across clusters in my ClusterSet with similarly named pods in each cluster. I need a way to disambiguate each backend pod via DNS.
 
-```
-<<[UNRESOLVED]>>
-Examples of DNS using cluster ID a la `<hostname>.<clusterID>.<svc>.<ns>.svc.clusterset.local`
-<<[/UNRESOLVED]>>
-```
+_For example, an exported headless service of services name `myservice` in namespace `test`,  backed by pods in two clusters with clusterIDs `clusterA` and `clusterB`, could be disambiguated by different DNS names following the pattern `<clusterID>.<svc>.<ns>.svc.clusterset.local`: `clusterA.myservice.test.svc.clusterset.local.` and `clusterB.myservice.test.svc.clusterset.local.`. This way the user can implement whatever load balancing they want (as is usually the case with headless services) by targeting each cluster's available backends directly._
 
 #### Diagnostics
 
 Clusters within my ClusterSet send logs/metrics to a common monitoring solution and I need to be able to identify the cluster from which a given set of events originated.
 
+#### Multi-tenant controllers
+
+My controller interacts with multiple clusters and needs to disambiguate between them to process its business logic.
+
+_For example, [CAPN's virtualcluster project](https://github.com/kubernetes-sigs/cluster-api-provider-nested) is implementing a multi-tenant scheduler that schedules tenant namespaces only in certain parent super clusters, and a separate syncer running in each parent super cluster controller needs to compare the name of the parent super cluster to determine whether the namespace should be synced. ([ref](https://github.com/kubernetes/enhancements/issues/2149#issuecomment-768486457))._
```

Comment:
is 'super cluster' intentional here, or is it just a parent cluster?

Reply:
It is intentional insofar as that is the terminology used by the cluster api nested WG, but it's not intentional in the way that `supercluster` used to be a term for the MCS API `clusterset`. I think it's safe for me to change this to just 'parent cluster' in this context to avoid confusion.

Reply:
Fixed in 6145f38ab48aba732e4f47261adb99b0d6723065